All Core Devs Meetings/Meeting 58.md

```diff
@@ -0,0 +1,298 @@
+# Ethereum Core Devs Meeting 58 Notes
+### Meeting Date/Time: Friday, March 29, 2019 14:00 UTC
+### Meeting Duration:  1 hours 45 minutes
+### [GitHub Agenda Page](https://github.com/ethereum/pm/issues/89)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=v8Psbo8zY4Y)
+### Moderator: Hudson Jameson
+
+# Summary
+
+### DECISIONS MADE
+
+**DECISION 58.1** The [ProgPoW Carbon Vote](http://www.progpowcarbonvote.com/) signal will be shutting down in 13 days from today. Everyone that wishes to vote will need to do so before this date and ensure they leave their ETH in the address they voted from until after the specified block number.
```

Comment:
This wasn't in the call, but Lane has since then defined the block number. May be worth including: https://twitter.com/lrettig/status/1111652965331415040

Reply:
Good call - updated.